libs/libc/wqueue/work_usrthread.c

```diff
@@ -281,7 +280,7 @@ int work_usrstart(void)
 
   /* Set up the work queue lock */
 
-  _SEM_INIT(&g_usrwork.lock, 0, 1);
+  nxmutex_init(&g_usrwork.lock);
```

Comment:
ditto. do we need to call `nxmutex_destroy` + `_SEM_DESTROY` if `task_create` fails

Reply:
next patch